src/client/css/partials/breaches.css

```diff
@@ -252,3 +256,41 @@
 .breaches-table details[open] > article {
   animation: fade-in 0.5s ease-out;
 }
+
+.breaches-table .no-unresolved-breaches-message > .no-breaches-message,
+.breaches-table .no-unresolved-breaches-message > .all-breaches-resolved-message {
+  display: none;
+}
+
+.breaches-table[data-nr-unresolved='0'][data-nr-resolved='0'] .no-unresolved-breaches-message .no-breaches-message {
+  display: flex;
+}
+
+.breaches-filter:not([data-selected='resolved']) + .breaches-table[data-nr-unresolved='0']:not([data-nr-resolved='0']) .no-unresolved-breaches-message .all-breaches-resolved-message {
```

Comment:
Nit (non-blocking): Would be nice to have a comment on what state we are targeting here.